Source/ActionController.swift

```diff
@@ -232,7 +291,17 @@ open class ActionController<ActionViewType: UICollectionViewCell, ActionDataType
         case .cellClass:
             collectionView.register(ActionViewType.self, forCellWithReuseIdentifier:ReusableViewIds.Cell.rawValue)
         }
-        
+
+        // register section cancel
+        if let cancelSpec = cancelSpec {
+            switch cancelSpec {
+            case .cellClass:
+                collectionView.register(CancelViewType.self, forCellWithReuseIdentifier: ReusableViewIds.Cancel.rawValue)
```

Comment:
I noticed something weird, the cancelView spec has a generic type that must be a subtype of `UICollectionReusableView` but here the cancelView is being registered as a cell. Why is that? Cannot you register the cancel view as a section footer?